### PR TITLE
benchkv: Make value size configurable

### PR DESCRIPTION
**Pull request by shenli**
@ngaut @siddontang @tiancaiamao @coocood  PTAL

### CODE REVIEW

File: benchkv/main.go [hunk 1]
```diff
@@ -93,11 +94,16 @@ func batchRW() {
 	wg.Wait()
 }
 
+var value []byte = []byte("value")
```
Comment (zimulala) [inline benchkv/main.go]: It can use a argument passed to `batchRW`.

File: benchkv/main.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ var (
 	dataCnt   = flag.Int("N", 1000000, "data num")
 	workerCnt = flag.Int("C", 400, "concurrent num")
 	pdAddr    = flag.String("pd", "localhost:2379", "pd address:localhost:2379")
+	valueSize = flag.Int("V", -1, "value size in byte")
```
Comment (coocood) [inline benchkv/main.go]: Why not use a positive size, and always use it to make value?

**Comment by shenli** [comment]
@zimulala PTAL

**Comment by zimulala** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Comment by disksing** [comment]
Please add license header for `main.go`.

**Comment by shenli** [comment]
@disksing PTAL

**Comment by disksing** [comment]
LGTM

**Pull request merged**